app/routes/fullPageExampleRoutes.js

```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+const fullPageExampleRouter = express.Router()
+const haveYouChangedYourNameController = require('../views/full-page-examples/have-you-changed-your-name')
+const feedbackController = require('../views/full-page-examples/feedback')
+const howDoYouWantToSignInController = require('../views/full-page-examples/how-do-you-want-to-sign-in')
+const passportDetailsController = require('../views/full-page-examples/passport-details')
+const updateYourAccountDetailsController = require('../views/full-page-examples/update-your-account-details')
+const uploadYourPhotoController = require('../views/full-page-examples/upload-your-photo')
+const whatIsYourAddressController = require('../views/full-page-examples/what-is-your-address')
+const whatIsYourNationalityController = require('../views/full-page-examples/what-is-your-nationality')
+const whatIsYourPostcodeController = require('../views/full-page-examples/what-is-your-postcode')
+const whatWasTheLastCountryController = require('../views/full-page-examples/what-was-the-last-country-you-visited')
+
+// All controllers needed for this Route
+const allFormControllers = [
+  haveYouChangedYourNameController,
+  feedbackController,
+  howDoYouWantToSignInController,
+  passportDetailsController,
+  updateYourAccountDetailsController,
+  uploadYourPhotoController,
+  whatIsYourAddressController,
+  whatIsYourNationalityController,
+  whatIsYourPostcodeController,
+  whatWasTheLastCountryController
+]
+
+// Display full page examples index by default if not handled already
+fullPageExampleRouter.route('/:example')
+  .get(function (req, res, next) {
+    res.render(`${req.params.example}/index`, function (error, html) {
+      if (error) {
+        next(error)
+      } else {
+        res.send(html)
+      }
+    })
+  })
+
+// Needed to perform validations when the forms are posted
+fullPageExampleRouter.use(allFormControllers)
```

Comment:
I know you haven't really touched this, but I'd be very tempted to get rid of the variables and the array and just require these files directly:

```
fullPageExampleRouter.use([
  require('../views/full-page-examples/have-you-changed-your-name')
  require('../views/full-page-examples/feedback')
  require('../views/full-page-examples/how-do-you-want-to-sign-in')
  require('../views/full-page-examples/passport-details')
  require('../views/full-page-examples/update-your-account-details')
  require('../views/full-page-examples/upload-your-photo')
  require('../views/full-page-examples/what-is-your-address')
  require('../views/full-page-examples/what-is-your-nationality')
  require('../views/full-page-examples/what-is-your-postcode')
  require('../views/full-page-examples/what-was-the-last-country-you-visited')
])
```